graphql/resolvers.go

```diff
@@ -67,6 +74,40 @@ func (r *hostResolver) Elapsed(ctx context.Context, obj *restModel.APIHost) (*ti
 	return obj.RunningTask.StartTime, nil
 }
 
+func (r *hostResolver) HomeVolumeDisplayName(ctx context.Context, obj *restModel.APIHost) (*string, error) {
+	if obj.HomeVolumeID == nil || *obj.HomeVolumeID == "" {
+		return nil, nil
+	}
+	volume, err := r.sc.FindVolumeById(*obj.HomeVolumeID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding volume %s: %s", *obj.HomeVolumeID, err.Error()))
+	}
+	if volume == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Unable to find volume %s", *obj.HomeVolumeID))
+	}
+	if volume.DisplayName == "" {
+		return obj.HomeVolumeID, nil
+	}
+	return &volume.DisplayName, nil
+}
+
+func (r *volumeResolver) HostDisplayName(ctx context.Context, obj *restModel.APIVolume) (*string, error) {
```

Comment:
wont it be easier to just return the full host object?

Reply:
yes, that's a great point